appservice/src/createAppService/AppServicePlanListStep.ts

```diff
@@ -80,8 +87,8 @@ export class AppServicePlanListStep extends AzureWizardPromptStep<IAppServiceWiz
             const isNewSiteLinux: boolean = wizardContext.newSiteOS === WebsiteOS.linux;
             let isPlanLinux: boolean = nonNullProp(plan, 'kind').toLowerCase().includes(WebsiteOS.linux);
 
-            if (plan.sku && plan.sku.family === 'EP') {
-                // elastic premium plans do not have the os in the kind, so we have to check the "reserved" property
+            if (plan.sku && (plan.sku.family === 'EP' || plan.sku.family === 'WS')) {
+                // elastic premium plans and workflow standard plans do not have the os in the kind, so we have to check the "reserved" property
                 // also, the "reserved" property is always "false" in the list of plans returned above. We have to perform a separate get on each plan
                 const client: WebSiteManagementClient = await createWebSiteClient(wizardContext);
                 const epPlan: WebSiteManagementModels.AppServicePlan | undefined = await tryGetAppServicePlan(client, nonNullProp(plan, 'resourceGroup'), nonNullProp(plan, 'name'));
```

Comment:
Will this work with Workflow plans still?  Are Workflow plans exclusively Linux or can they be Windows as well?

Reply:
Workflow standard sku is exactly similar to elastic plan, bit that is only specific to logic apps. Now currently these only support windows.